doc/API.md

```diff
@@ -93,7 +93,7 @@ Allow to initiate the box by registering an admin user. CORS is not allowed for
 ### Request
 ___Parameters___
 * email - Admin email.
-* username - User name. Defaults to "admin".
+* username - Optional. User name. Defaults to "admin".
```

Comment:
If we are going to make this optional, I would rather remove it, and leave just the email as a way of authenticating.